riscv/src/large_field/bootloader.rs

```diff
@@ -14,7 +14,7 @@ pub fn bootloader_preamble() -> String {
     let mut preamble = r#"
     // ============== bootloader-specific instructions =======================
     // Write-once memory
-    std::machines::write_once_memory::WriteOnceMemory bootloader_inputs;
+    std::machines::write_once_memory::WriteOnceMemory bootloader_inputs(MIN_DEGREE, MAIN_MAX_DEGREE);
```

Comment:
Interesting! I guess this makes sense because we might have different number of inputs in each chunk.

Reply:
I mean, we could definitely make the max smaller, but I figured it's safe and not too large anyway